docs/configuration.md

```diff
@@ -8,7 +8,7 @@ Global configuration settings override system configuration settings, and local
 GCM Core honors several levels of settings, in addition to the standard local \> global \> system tiering Git uses.
 URL-specific settings or overrides can be applied to any value in the `credential` namespace with the syntax below.
 
-Additionally, GCM Core respects several GCM-specific [environment variables](environment.md) **which take precedence over configuration options.**
+Additionally, GCM Core respects several GCM-specific [environment variables](environment.md) **which take precedence over configuration options**. System administrators may also configure [default values](enterprise-config.md) for many settings used by GCM Core.
```

Comment:
Should we make it _really_ explicit here that the defaults are overridden by config values and then environment variables?

```suggestion
GCM Core has multiple sources that provide customization of configured values. The following sources are used, in priority order:

1. GCM-specific [environment variables](environment.md)
2. Values available in `git config`. Git prioritizes local config settings over global and system config settings.
3. [System administrator-set default values](enterprise-config.md).
4. Compiled defaults in the GCM codebase.
```

Not sure this is better, but it at least makes the order very clear. Also, the two preceding paragraphs would want edits to work more closely with this format.

Reply:
I don't think we need to be explicit here and present a list. We're presenting a list in the `enterprise-config.md` file, and the text in `configuration.md` does say these are "default values" only so that shouldn't cause confusion or concern.